skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs/CoNLL-2000/README

```diff
@@ -0,0 +1,13 @@
+Examples from CoNLL 2000 shared task: Chunking.
```

Comment:
Do we need these folders: 

- `skema/text_reading/text_reading/webapp/public/brat/static/example-READMEs`
- `skema/text_reading/text_reading/webapp/public/brat/static/case-studies`
- `skema/text_reading/text_reading/input/grfn`

Reply:
If not, can we get rid of them?

Reply:
We don't need them, I removed them